attributes/network/cddl/6_ledger_commands.cddl

```diff
@@ -1,32 +1,5 @@
-; tag::mint[]
```

Comment:
I don't think we want to remove those! `mint` and `burn` are nowhere to be found!

Reply:
We're removing this, either as part of this or another PR. Let me make another PR first I guess.

Reply:
https://github.com/many-protocol/specification/pull/20

Reply:
It won't merge nicely but I'll manage the merge conflict manually.